lib/raster3d/open.c

```diff
@@ -152,7 +152,7 @@ void *Rast3d_open_cell_old(const char *name, const char *mapset,
 	    Rast3d_error(_("Rast3d_open_cell_old: can't read header"));
 	    return (void *)NULL;
 	}
-	Rast3d_long_decode(ltmp, &(map->indexOffset), 1, map->indexLongNbytes);
+	Rast3d_long_decode((unsigned char *)ltmp, &(map->indexOffset), 1, map->indexLongNbytes);
```

Comment:
Change type of ltmp at the beginning of `Rast3d_open_cell_old()`?

Reply:
In this case it seems to work to define `unsigned char *ltmp;`.

Reply:
OK. Again, seems to be an inconsistency in the raster3d lib.

Reply:
Updated according to suggestion.